src/main.rs

```diff
@@ -0,0 +1,14 @@
+use std::io;
+
+fn main() {
+    let mut name = String::new();
+    println!("Please inout your name.");
+    io::stdin()
```

Comment:
By convention, in most cases we like to include complete `use `paths as it makes the code cleaner and easier to read. I.e.:
```suggestion
use std::io::stdin;
...
    stdin()
```

Reply:
fixed it